### PR TITLE
REVIEW: XMPP Newsletter August 2021

### PR DESCRIPTION
**Pull request by wurstsalat3000**
### I created the draft for the next release covering **August 2021**. 

@wurstsalat3000 is helping out @Echolon this month.

### Final release is planned for **05 of September 2021**.

---

You can make a simple comment or create a merge request to make things easy going for review. You don't need any special technical knowledge to contribute. If you need help, please ask for help in the Communication-Team MUC: commteam@muc.xmpp.org

- [You can use our online Pad](https://yopad.eu/p/xmpp-newsletter-365days)! Please be tidy and write something instead of dropping bare HTTP links :pray: 
- Just paste your prepared text in this thread below or in our XMPP channel (xmpp:commteam@muc.xmpp.org?join)
- [Make a pull request by editing the file in this repository](https://github.com/xsf/xmpp.org/blob/newsletter-august2021/content/posts/newsletter/2021-09-05.md)

:arrow_right: Please do not just drop bare links - rather try to make a short and decent formulated description (**<5 lines**) what is the main outcome of your source. You do not need to write in a hard, formal tone - be passionate why this motivates you or what you work on! 

**Note: If you create a pull request, which is highly appreciated, please @mention me once your PR is created!** :pray: 

[Insert timeline for September here.]

---

This is our planed timeline for this draft:

Status:
- [x] Collection of news
- [x] Review
- [ ] Blog post + Tiny Newsletter
- [ ] Social media
- [ ] Translations
- [ ] Archived ([Use pandoc!](https://pandoc.org))

### CODE REVIEW

**Comment by Echolon** [comment]
And it is great that @wurstsalat3000 is helping out. Thanks a lot!!!! great support!

File: content/posts/newsletter/2021-09-05.md [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+
+While the Snikket iOS client app was just released, read more below, the behind the scenes development continues. In the latest blog post, Matthew Wild announces that [the expert folk at Simply Secure will be performing a usability audit of the current app](https://snikket.org/blog/simply-secure-collaboration/), as well as conducting usability testing thanks to funding from the OTF’s Usability Lab. The analysis will help to improve the UX of the iOS app and Snikket as a whole.
+
+Communiqué is a new XMPP client from the [Mellium Co-op](https://opencollective.com/mellium) team. It was [announced](https://opencollective.com/mellium/updates/introducing-communique) this month and presented at the XMPP Office Hours (TODO: add video link). The source code can be found [in the repository](https://github.com/mellium/xmpp/). It includes updates to the Mellium XMPP library as well as the new Communiqué instant messaging client (discussed more below). The biggest updates this month are MAM and ad-hoc commands support.
```
Comment (SamWhited) [inline content/posts/newsletter/2021-09-05.md:68]: The TODO in here needs removing

**Comment by SamWhited** [comment]
The Mellium library updates seem to have gotten dropped between the pad and here?

File: content/posts/newsletter/2021-09-05.md [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+
+In [the previous newsletter](https://xmpp.org/2021/07/newsletter-07-july/) we mentioned that Debian Linux 11 will soon be launched with updated XMPP software, as this happened in the meantime, server admins are already updating or even setting up new deployments. Such as Nelson from Luxembourg, who published a blog post about setting up a [server with ejabberd on Debian 11 Bullseye](https://www.aroundtheglobe.biz/posts/20210819-your_own_xmpp_server_with_ejabberd_on_Debian_11_Bullseye.html).
+
+Missed in last months issue, the folks at [cometchat](https://www.cometchat.com/) have blogged about XMPP's history, working architecture, stanzas and
```
Comment (wurstsalat3000) [inline content/posts/newsletter/2021-09-05.md:47]: last **month's**

File: content/posts/newsletter/2021-09-05.md [hunk 3]
```diff
@@ -0,0 +1,193 @@
...
+features in general in [Everything About XMPP - Extensible Messaging & Presence Protocol](https://www.cometchat.com/blog/xmpp-extensible-messaging-presence-protocol). If you want a quick technical overview (or need one to show others what XMPP is all about) this ~15 minutes read can bring one up to speed.
+
+"Spaces" are the new XMPP frontier to be explored, and you'd get a glimpse of them in Gajim client news below, but the work is pretty elaborate and ongoing with many people involved. [Renga](https://github.com/HaikuArchives/Renga)'s (an XMPP client for Haiku) developer [pulkomandy](https://pulkomandy.tk/_/_.users/_pulkomandy) has blogged [Some random thoughts about XMPP spaces](https://pulkomandy.tk/_/_Development/_Some%20random%20thoughts%20about%20XMPP%20spaces) thinking about use cases (family, business, communities) and user interfaces.
+
+Any Turkish speakers reading the newsletter? We don't have a translation yet, but [Ged](https://fosstodon.org/@Ged) has just published an in-depth blog post about XMPP called [Hangi "Chat" Programı?](https://www.yunbere.kalfazade.com/post/hangi-chat-programi). In about 40 minutes it takes the reader through the story of the
```
Comment (wurstsalat3000) [inline content/posts/newsletter/2021-09-05.md:52]: post about XMPP **titled**

File: content/posts/newsletter/2021-09-05.md [hunk 4]
```diff
@@ -0,0 +1,193 @@
...
+-   Version 1.24.0 of [XEP-0001](https://xmpp.org/extensions/xep-0001.html) (XMPP Extension Protocols)
+    -   Change "Draft" to "Stable". (ssw)
+
+### Last Call
+
+Last calls are issued once everyone seems satisfied with the current XEP status. After the Council decides whether the XEP seems ready, the XMPP Editor issues a Last Call for comments. The feedback gathered during the Last Call help improving the XEP before returning it to the Council for advancement to Draft.
+
+-   No Last Call this month.
+
+### Draft
```
Comment (wurstsalat3000) [inline content/posts/newsletter/2021-09-05.md:135]: Stable

Comment (licaon-kter) [reply content/posts/newsletter/2021-09-05.md:135]: Maybe `### Stable (formely known as Draft)` makes is clearer? And from September it's only 'Stable'

Comment (wurstsalat3000) [reply content/posts/newsletter/2021-09-05.md:135]: Sounds good to me!

Comment (Echolon) [reply content/posts/newsletter/2021-09-05.md]: 7c930db

File: content/posts/newsletter/2021-09-05.md (hunk 3 above)
Comment (Echolon) [reply content/posts/newsletter/2021-09-05.md]: 7c930db

File: content/posts/newsletter/2021-09-05.md (hunk 2 above)
Comment (Echolon) [reply content/posts/newsletter/2021-09-05.md]: 7c930db

File: content/posts/newsletter/2021-09-05.md (hunk 1 above)
Comment (Echolon) [reply content/posts/newsletter/2021-09-05.md]: 7c930db

File: content/posts/newsletter/2021-09-05.md [hunk 5]
```diff
@@ -0,0 +1,197 @@
...
+ 
+Image: App store
+
+## Servers
+
+Prosody 0.11.10 has been [released](https://blog.prosody.im/prosody-0.11.10-released/) with a fix for CVE-2021-37601 and some minor changes. Prosody developers recommend server admins to upgrade in order to fix the remote information disclosure issue.
+
+## Libraries
+
+The Mellium Dev Communiqué for August includes updates to the Mellium XMPP library as well as the new Communiqué instant messaging client (discussed more below). The biggest updates this month are MAM and ad-hoc commands support! You can read more here:  https://opencollective.com/mellium/updates/dev-communique-for-august-2021
```
Comment (SamWhited) [inline content/posts/newsletter/2021-09-05.md:88]: "discussed more above" maybe? I don't know if it got shuffled around or I mixed things up somehow before, but this doesn't seem right. Thanks for fixing it and adding it back!

**Review by Echolon — APPROVED** [review]
Approved for release!

File: content/posts/newsletter/2021-09-05.md (hunk 5 above)
Comment (Echolon) [reply content/posts/newsletter/2021-09-05.md]: 4c003d0

**Comment by Echolon** [comment]
:hourglass:  **Editorial closing of the XMPP Newsletter August release**
:rocket: Merge in a few minutes! Docker build pulls it asap :robot:
:pray: Thanks to Adrien Bourmault (neox), Anoxinon e.V. community, anubis, Benoît Sibaud, emus, Sam, Licaon_Kter, nicola, seveso, SouL, wurstsalat3000, Ysabeau for their help in creation, review and translation!
:handshake: ... ehh.. here we go again, https://timetoherd.com/ :hand: :mask: :hand:
:heart: :fireworks: **Next release on 5th of October 2021**

**Pull request merged**